.env.test

```diff
@@ -0,0 +1,9 @@
+# Dotenv
+#
+# This file commits safe environment variables for the test environment.
+# For managing sensitive values and overrides use `/.env.test.local`
+#
+# Reference: https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
+
+# TODO: Replace `rails-template` with the name of the app.
+DATABASE_URL=postgres://postgres@localhost:5432/dfsseta-apply-for-landing-test
```

Comment:
I think this TODO is TODONE, so that comment can probably go?

Reply:
thks, good point!